contracts/CourtPresaleActivate.sol

```diff
@@ -153,7 +161,11 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
 
         // activate in registry
         bondedToken.approve(address(registry), bondedTokensObtained);
-        registry.stakeFor(_from, bondedTokensObtained, abi.encodePacked(ACTIVATE_DATA));
+        bytes memory data;
```

Comment:
```suggestion
        bytes memory data = new bytes(0);
```

Otherwise we're passing an invalid memory pointer around.

Reply:
Good catch! So (badly) used to not initialize variables in Solidity... Thanks!